src/components/Profile.tsx

```diff
@@ -0,0 +1,329 @@
+import { Form, Button, Offcanvas, CloseButton } from 'react-bootstrap';
```

Comment:
At which route I can look for this component?

Reply:
This component will be added to the Navbar of the game,which is triggered upon a button click

Reply:
So how do i check how its looking? 
Is it in storybook or can you send a ss?